legacy/PimExportHelper.js

```diff
@@ -43,15 +43,21 @@ class PimExportHelper {
    * @param {string} fieldApi
    */
   getValue(object, fieldApi) {
+    if (!object || !fieldApi) {
```

Comment:
I would change this to
```
if (object === null || fieldApi === null)...
```
this covers empty as well as undefined.

Reply:
not following. We do want to check empty or undefined, right?

Reply:
never mind I said this comment.